clouddriver-titus/src/main/groovy/com/netflix/spinnaker/clouddriver/titus/deploy/actions/DeleteTitusScalingPolicy.java

```diff
@@ -53,10 +60,12 @@ public Result apply(
       @NotNull DeleteTitusScalingPolicy.DeleteTitusScalingPolicyCommand command,
       @NotNull Saga saga) {
     saga.log("Initializing Delete Scaling Policy " + command.description.getScalingPolicyID());
+    AccountCredentials accountCredentials =
+        accountCredentialsRepository.getOne(command.description.getAccount());
 
     TitusAutoscalingClient client =
         titusClientProvider.getTitusAutoscalingClient(
-            command.description.getCredentials(), command.description.getRegion());
```

Comment:
command.description.getCredentials() was returning `NULL` for some reason , updated to fetch the credentials before creating the titus client